{"uid":"e1d9f2853dff0320","name":"test_smoke","fullName":"tests.integration.test_smoke#test_smoke","historyId":"78ffebf457376473b3476ee0f7a454c9","time":{"start":1732152156779,"stop":1732152894856,"duration":738077},"description":"Deploy bundle with app and release as a product.","descriptionHtml":"<p>Deploy bundle with app and release as a product.</p>\n","status":"failed","statusMessage":"AssertionError: assert 'mysql-database' in ['information_schema', 'mysql', 'performance_schema', 'sys']","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/tests/integration/test_smoke.py\", line 110, in test_smoke\n    assert \"mysql-database\" in databases\nAssertionError: assert 'mysql-database' in ['information_schema', 'mysql', 'performance_schema', 'sys']","flaky":true,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"tmp_path_factory","time":{"start":1732152156647,"stop":1732152156647,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop","time":{"start":1732152156647,"stop":1732152156647,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1732152156648,"stop":1732152156778,"duration":130},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1732152156778,"stop":1732152156778,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Deploy bundle with app and release as a product.","status":"failed","statusMessage":"AssertionError: assert 'mysql-database' in ['information_schema', 'mysql', 'performance_schema', 'sys']","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/mysql-bundle/mysql-bundle/tests/integration/test_smoke.py\", line 110, in test_smoke\n    assert \"mysql-database\" in databases\nAssertionError: assert 'mysql-database' in ['information_schema', 'mysql', 'performance_schema', 'sys']","steps":[],"attachments":[{"uid":"403056c1ae008b69","name":"log","source":"403056c1ae008b69.txt","type":"text/plain","size":194630}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1732152894896,"stop":1732152894897,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1732152894897,"stop":1732152896721,"duration":1824},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1732152896722,"stop":1732152896722,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1732152896722,"stop":1732152896722,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1732152896722,"stop":1732152896722,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_smoke"},{"name":"host","value":"fv-az1117-567"},{"name":"thread","value":"15888-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_smoke"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"979de1221fbe39eb","status":"unknown","time":{}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":15,"broken":6,"skipped":0,"passed":86,"unknown":3,"total":110},"items":[{"uid":"8446d27143923545","reportUrl":"../1325//#testresult/8446d27143923545","status":"passed","time":{"start":1732065785082,"stop":1732066645410,"duration":860328}},{"uid":"2a4602c3bd28e5b2","reportUrl":"../1323//#testresult/2a4602c3bd28e5b2","status":"failed","statusDetails":"AssertionError: assert 'mysql-database' in ['information_schema', 'mysql', 'performance_schema', 'sys']","time":{"start":1731979366927,"stop":1731980115659,"duration":748732}},{"uid":"35d87420c5d52234","reportUrl":"../1319//#testresult/35d87420c5d52234","status":"failed","statusDetails":"AssertionError: assert 'mysql-database' in ['information_schema', 'mysql', 'performance_schema', 'sys']","time":{"start":1731892982241,"stop":1731893714118,"duration":731877}},{"uid":"1505a716b67765","reportUrl":"../1317//#testresult/1505a716b67765","status":"passed","time":{"start":1731806615071,"stop":1731807593390,"duration":978319}},{"uid":"bfa91c4843da08bb","reportUrl":"../1315//#testresult/bfa91c4843da08bb","status":"failed","statusDetails":"AssertionError: assert 'mysql-database' in ['information_schema', 'mysql', 'performance_schema', 'sys']","time":{"start":1731720296032,"stop":1731721192306,"duration":896274}},{"uid":"7a266448ab699a1d","reportUrl":"../1313//#testresult/7a266448ab699a1d","status":"passed","time":{"start":1731633802461,"stop":1731634737183,"duration":934722}},{"uid":"7f02ad1413cacb7e","reportUrl":"../1311//#testresult/7f02ad1413cacb7e","status":"passed","time":{"start":1731547393456,"stop":1731548241482,"duration":848026}},{"uid":"e7478acf18c75fd9","reportUrl":"../1309//#testresult/e7478acf18c75fd9","status":"failed","statusDetails":"AssertionError: assert 'mysql-database' in ['information_schema', 'mysql', 'performance_schema', 'sys']","time":{"start":1731460985234,"stop":1731461765088,"duration":779854}},{"uid":"c4de833afeab799d","reportUrl":"../1306//#testresult/c4de833afeab799d","status":"broken","statusDetails":"mysql.connector.errors.DatabaseError: 2003 (HY000): Can't connect to MySQL server on '10.9.56.228:3306' (111)","time":{"start":1731374537031,"stop":1731375226985,"duration":689954}},{"uid":"c40e3107d59b2e6","reportUrl":"../1303//#testresult/c40e3107d59b2e6","status":"passed","time":{"start":1731288184192,"stop":1731288976366,"duration":792174}},{"uid":"a481aaa541a0d4e6","reportUrl":"../1301//#testresult/a481aaa541a0d4e6","status":"passed","time":{"start":1731201791466,"stop":1731202611785,"duration":820319}},{"uid":"77ae8233044f5875","reportUrl":"../1299//#testresult/77ae8233044f5875","status":"passed","time":{"start":1731115361646,"stop":1731116284907,"duration":923261}},{"uid":"56e55db5cb167c01","reportUrl":"../1297//#testresult/56e55db5cb167c01","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1731028950224,"stop":1731030168169,"duration":1217945}},{"uid":"8423c91e3f3cf28a","reportUrl":"../1295//#testresult/8423c91e3f3cf28a","status":"passed","time":{"start":1730942530473,"stop":1730943452213,"duration":921740}},{"uid":"e3fa8e3e83da64b6","reportUrl":"../1293//#testresult/e3fa8e3e83da64b6","status":"failed","statusDetails":"AssertionError: assert 'mysql-database' in ['information_schema', 'mysql', 'performance_schema', 'sys']","time":{"start":1730856141647,"stop":1730856872684,"duration":731037}},{"uid":"1657a88401d978c1","reportUrl":"../1291//#testresult/1657a88401d978c1","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError","time":{"start":1730769769809,"stop":1730770993222,"duration":1223413}},{"uid":"48c1fcf987474e3a","reportUrl":"../1288//#testresult/48c1fcf987474e3a","status":"failed","statusDetails":"AssertionError: assert 'mysql-database' in ['information_schema', 'mysql', 'performance_schema', 'sys']","time":{"start":1730683350699,"stop":1730684074909,"duration":724210}},{"uid":"20ad28493765ddc9","reportUrl":"../1286//#testresult/20ad28493765ddc9","status":"passed","time":{"start":1730596957136,"stop":1730597803611,"duration":846475}},{"uid":"2dadeb2b63303257","reportUrl":"../1282//#testresult/2dadeb2b63303257","status":"passed","time":{"start":1730424326989,"stop":1730426073668,"duration":1746679}},{"uid":"5e3717f6d9d0371f","reportUrl":"../1280//#testresult/5e3717f6d9d0371f","status":"passed","time":{"start":1730337751897,"stop":1730338676412,"duration":924515}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"e1d9f2853dff0320.json","parameterValues":[]}